tests/test_cli.py

```diff
@@ -44,24 +46,29 @@ def assert_expected_file(self, result, output_path):
         except:
             pass
 
-    def test_schema_convert_cli(self, runner, helpers):
-        data_model_csv_path = helpers.get_data_path("example.model.csv")
-
-        output_path = helpers.get_data_path("example.model.jsonld")
-
-        label_type = 'class_label'
+    @pytest.mark.parametrize(
+        "model, output, expected",
+        [
+            ("tests/data/example.model.csv", "tests/data/example.model.pickle", 0),
+            ("tests/data/example.model.csv", "tests/data/example.model.jsonld", 0),
+        ],
+    )
+    def test_schema_convert_cli(self, runner, helpers, model, output, expected):
```

Comment:
can you also test the situation where you are explicitly passing the output_type parameter instead of using the path to infer the output type?
You could call parameterize several times:
Once pass just the example model
Next pass both `--output_jsonld` with different paths
Next pass` --output_type`, and account for all output types.

Doing it this way will test a bunch of combinations.

Then you could call the runner in different combos
1. with `output_jsonld`
2. with `output_type`
3. with both

What would you expect to happen if someone passed a `.pickle` `--output_jsonld` but then said the output type was graph? Vice versa?

I know this might be a lot, we could leave it for now and create an additional ticket.

Reply:
Would also want to pass none for these variables to ensure the defaults work as expected

Reply:
@mialy-defelice took me a minute to figure out parametrize, but how does this look? It's still a WIP since I haven't implemented all the scenarios you mentioned, but I want to make sure this is the structure you imagined.

Reply:
So yes, it looks right in that youre calling parameterize twice, but I would drop some of the tuples, bc some of the data repeats you dont need it:
```
    @pytest.mark.parametrize(
        "output_path",
        [
            "tests/data/example.model.pickle",
            "tests/data/example.model.jsonld",
        ],
        ids=["output_path_pickle", "output_path_jsonld"],
    )
    @pytest.mark.parametrize(
        "output_type",
        [
            "jsonld",
            "graph",
            "all"
        ],
        ids=["output_type_jsonld", "output_type_graph", "output_type_all"],
    )
    def test_schema_convert_cli(self, runner, output_path, output_type):
        model = "tests/data/example.model.csv"
        expected = 0
        ```